### PR TITLE
implemented verify commands

### LINKED ISSUE #17: Setup moderation commands and interface with the db
Needed:
- [x] /setModeratorRole
- [x] /setAlliesRole
- [x] /setAxisRole
- [x] /setSpectatorRole
- [ ] /verifyAll
- [x] /setLogChannel
- [x] /setVerifyChannel
- [x] /setTroopMovementChannel
- [x] /setFacilityChannel
- [x] /addCommandChannel

### PR DESCRIPTION
**Pull request by TotallyNot**
resolves #17

### CODE REVIEW

File: src/wrapper/models/user.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Statistics, Timers } from "./common";
 export interface UserData {
     id: number;
     name: string;
-    team: string;
+    team: "Allies" | "Axis" | "None";
```
Comment (Natty-Boh) [inline src/wrapper/models/user.ts:6]: auto?

Comment (TotallyNot) [reply src/wrapper/models/user.ts:6]: isn't in the api to my best knowledge

Comment (Natty-Boh) [reply src/wrapper/models/user.ts:6]: im like 95% sure it is...

File: src/handler/guild/verifyAll.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+import { BotError, ApiError } from "../../error";
+
+import { Guild } from "../../entity/Guild";
+
+export class Verify {
+    @CommandData({ type: "guild" })
+    verifyAllData() {
+        return new SlashCommandBuilder()
+            .setName("verify-all")
+            .setDescription("Verify user")
```
Comment (Natty-Boh) [inline src/handler/guild/verifyAll.ts:18]: this isnt a very good description...

Comment (TotallyNot) [reply src/handler/guild/verifyAll.ts:18]: it's... copy pasted

**Review by Natty-Boh — APPROVED** [review]

**Pull request merged**